config/prow-staging/cluster/BUILD.bazel

```diff
@@ -10,6 +10,21 @@ load("//def:configmap.bzl", "k8s_configmap")
 
 release(
     "staging",
+    component(
+        "boskos",
```

Comment:
What kind of boskos problems will this detect?

Reply:
I'm planning to add some smoke tests for `boskos staging` together with `prow staging`. At least it should be able to test:
- If the Boskos service can be started, by checking if all the Boskos Pods are running.
- If the `acquire` and `release` APIs are still functioning.

Though we may never need them if https://github.com/kubernetes-sigs/boskos has enough e2e test coverage 🤔